docker-compose.yml

```diff
@@ -39,7 +39,7 @@ nginx:
     - web:web
 
 memcached:
-  image: memcached
+  image: memcached:1.4.24
```

Comment:
If you pin versions to this degree it does mean we don't get updates without manual intervention. E.g. if a security patch came out in `1.4.25` or `1.5` we won't get it.